src/gluonts/model/rotbaum/_predictor.py

```diff
@@ -119,9 +121,10 @@ def __init__(
         clump_size: int = 100,  # Used only for "QRX" method.
         context_length: Optional[int] = None,
         use_feat_static_real: bool = False,
-        use_feat_static_cat: bool = False,
         use_feat_dynamic_real: bool = False,
         use_feat_dynamic_cat: bool = False,
+        cardinality="auto",
```

Comment:
I think we want to still use type annotations here.

You can use `Union[List[int], ...]` to allow more than one type.